docs/tutorials/vqe_with_estimator.ipynb

```diff
@@ -407,38 +385,110 @@
     "    \"prev_vector\": None,\n",
     "    \"iters\": 0,\n",
     "    \"cost_history\": [],\n",
-    "    \"_total_time\": 0,\n",
-    "    \"_prev_time\": None,\n",
     "}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now free to use a classical optimizer of our choice to minimize our cost function. Here we use the [COBYLA routine from SciPy through the `minimize` function](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.minimize.html). Note that, when running on real quantum hardware, the choice of optimizer is important as not all optimizers handle noisy cost function landscapes equally well.\n",
+    "\n",
+    "To begin the routine, we start by specifying a random initial set of parameters:"
+   ]
+  },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x0 = 2 * np.pi * np.random.random(num_params)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Because we are sending a large number of jobs that we would like to execute together, here we use a [`Session`](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Session) to execute all the generated circuits in one block.  Here `args` is the standard SciPy way to supply the additional parameters needed by the cost function."
```

Comment:
```suggestion
    "Because we are sending a large number of jobs that we would like to execute together, we use a [`Session`](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Session) to execute all the generated circuits in one block.  Here `args` is the standard SciPy way to supply the additional parameters needed by the cost function."
```